### PR TITLE
Mobile app rewrite NR-51392

### PR DESCRIPTION
**Pull request by austin-schaefer**
Still to do:

- [x] Flatten IA
- [x] Add widget screenshot and link to main doc
- [x] Add screenshot to Android docs
- [x] Add screenshot to iOS doc
- [x] iOS widgets doc screenshot import statement update
- [x] Mobile apps team review of copy to ensure it's accurate

Reviewer notes:

* I rewrote the Android/iOS docs pretty much completely
* The new copy might be a bit too marketing-y or janky
* Other changes were mostly flattening taxonomy

### CODE REVIEW

File: src/content/docs/mobile-apps/android.mdx [hunk 1]
```diff
@@ -14,14 +14,9 @@ redirects:
...
-import iconIosClock0 from 'images/icon-ios-clock_0.png'
+import mobileappsAndroidAppAPMDashboard from 'images/mobile-apps_screenshot-full_Android-app-APM-dashboard.png'
 
-The user interface for New Relic's Android app provides functionality similar to New Relic's standard user interface, with customized details for mobile users. Learn more from the [New Relic website](https://newrelic.com/mobility). 
-
-## Compatibility and requirements [#reqs]
-
-* Android 8.0 (Oreo) or higher
-* Screen size of 7 inches or less
+New Relic's Android app lets you access your data wherever you are. You can view curated dashboards from across New Relic capabilities, from APM to infrastructure to browser, and access any dashboards you've built. If you need to go deeper, you can query your data directly with NRQL or search your logs. And you can receive alerts directly on the app so you can start investigating immediately.
```
Comment (bradleycamacho) [inline src/content/docs/mobile-apps/android.mdx:19]: Sat in on many calls between the VM team and legal. They (legal) really want us avoiding  `New Relic's` wherever possible. I'd replace all instances of `New Relic's Android app` w/ `The New Relic Android app`

I'll mark all places I find it with  `Here`

File: src/content/docs/mobile-apps/android.mdx [hunk 2]
```diff
@@ -4,7 +4,7 @@ tags:
   - Mobile apps
```
Comment (bradleycamacho) [inline src/content/docs/mobile-apps/android.mdx:4]: Won't let me add a comment on the right line, but I think there should be a `the` between `to` and `New Relic`

File: src/content/docs/mobile-apps/android.mdx [hunk 3]
```diff
@@ -40,17 +35,27 @@ The user interface for New Relic's Android app provides functionality similar to
   Depending on your New Relic account, you may need to [further authenticate your user](/docs/mobile-apps/new-relic-mobile-apps/authentication-alerts/user-settings-authentication).
 </Callout>
 
-## Features [#features]
-
-The New Relic Android app includes data about these features:
+<SideBySide>
+  <Side>
+    ## New Relic capabilities [#features]
```
Comment (bradleycamacho) [inline src/content/docs/mobile-apps/android.mdx:40]: Explore adding links to these specific categories unless you think that'd be too much link fatigue.

File: src/content/docs/mobile-apps/android.mdx (hunk 1 above)
Comment (bradleycamacho) [reply src/content/docs/mobile-apps/android.mdx:19]: Style guide reference: https://docs.newrelic.com/docs/style-guide/capitalization/product-capability-feature-usage/

File: src/content/docs/mobile-apps/android.mdx [hunk 4]
```diff
@@ -40,17 +35,27 @@ The user interface for New Relic's Android app provides functionality similar to
...
-  icon to see detailed transaction metrics, or an **Overview** chart to view summary charts of your top five transactions. Select the <Icon name="fe-filter"/>
-  icon to filter by [labels and categories](/docs/data-analysis/user-interface-functions/labels-categories-organize-your-apps-servers).
-* Browser monitoring metrics, including average page load time, Apdex, average throughput, and more.
-* Infrastructure monitoring.
-* Mobile monitoring, including [crash reports](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-dashboards/mobile-apps-crash-dashboard), network errors, API calls, and active user count.
-* Event notifications, including [mobile alerts](/docs/mobile-apps/new-relic-ios-app/features/ios-alerting) wherever you are, plus [deployment notifications](/docs/applications-menu/deployments-dashboard) and [notes](/docs/site/public-and-private-notes).
+    * **Alerts**: View current and past incidents, acknowledge incidents, and tap through to see performance data for related entities.
+    * **APM**: See real-time and historical data across transactions, errors, databases, external services, and more.
```
Comment (bradleycamacho) [inline src/content/docs/mobile-apps/android.mdx:43]:
```suggestion
    * **Application performance monitoring (APM)**: See real-time and historical data across transactions, errors, databases, external services, and more.
```

Comment (bradleycamacho) [reply src/content/docs/mobile-apps/android.mdx:43]: Should technically be expanded in intro, but I think that would make it a bit too complex of a sentence.

File: src/content/docs/mobile-apps/android.mdx [hunk 5]
```diff
@@ -40,17 +35,27 @@ The user interface for New Relic's Android app provides functionality similar to
...
+    * **Browser monitoring**: Track throughput, load times, errors, AJAX requests and more across all your front-end web apps.
+    * **Dashboards and NRQL**: View your dashboards, or dive deeper with NRQL. You can build custom NRQL queries, or you can view the NRQL for any chart by tapping on the chart, then tapping the overflow <Icon name="fe-more-horizontal" /> icon and selecting **View query**.
+    * **Explorer**: Browse everything else in New Relic, from AWS services to network performance.
+    * **Infrastructure monitoring**: See the health of your hosts.
```
Comment (bradleycamacho) [inline src/content/docs/mobile-apps/android.mdx:47]: Seems weak compared to the rest of the bullets. Yoinking something from the intro to infra doc could help:

`**Infrastructure monitoring**: See the health of your hosts running in the cloud, on dedicated hosts, or containers running in orchestrated environments.`

File: src/content/docs/mobile-apps/ios-app/ios.mdx [hunk 6]
```diff
@@ -20,16 +20,9 @@ redirects:
...
 
-The New Relic iOS app allows you to view your New Relic applications, infrastructure data, key transactions, synthetic monitors, and alerts from an iPhone or iPad.  Learn more from the [New Relic website](https://newrelic.com/mobility). 
-
-## Compatibility and requirements [#reqs]
-
-* iOS 13 or higher
-* iPhone SE or higher
-* iPad Air 2 or higher
-* iPod touch. You can use it, but screen resolution may be different.
+New Relic's iOS app lets you access your data wherever you are. You can view curated dashboards from across New Relic capabilities, from APM to infrastructure to browser, and access any dashboards you've built. If you need to go deeper, you can query your data directly with NRQL or search your logs. And you can receive alerts directly on the app so you can start investigating immediately.
```
Comment (bradleycamacho) [inline src/content/docs/mobile-apps/ios-app/ios.mdx:25]:
```suggestion
The New Relic iOS app lets you access your data wherever you are. You can view curated dashboards from across New Relic capabilities, from APM to infrastructure to browser, and access any dashboards you've built. If you need to go deeper, you can query your data directly with NRQL or search your logs. And you can receive alerts directly on the app so you can start investigating immediately.
```

File: src/content/docs/mobile-apps/ios-app/ios.mdx [hunk 7]
```diff
@@ -48,16 +41,29 @@ The New Relic iOS app allows you to view your New Relic applications, infrastruc
   Depending on your New Relic account, you may need to [further authenticate your user](/docs/mobile-apps/new-relic-mobile-apps/authentication-alerts/user-settings-authentication).
 </Callout>
 
-## Features [#features]
-
-New Relic's iOS app includes these New Relic products and features:
+<SideBySide>
+  <Side>
+    ## New Relic capabilities [#features]
```
Comment (bradleycamacho) [inline src/content/docs/mobile-apps/ios-app/ios.mdx:46]: Same comments as the android app about linking and expanding infra bullet

**Review by bradleycamacho — CHANGES_REQUESTED** [review]

File: src/content/docs/mobile-apps/android.mdx (hunk 3 above)
Comment (austin-schaefer) [reply src/content/docs/mobile-apps/android.mdx:40]: We originally had lnks but my sense was that thye're not particularly helpful in this case and just add to link fatigue.

**Review by bradleycamacho — DISMISSED** [review]

**Review by bradleycamacho — APPROVED** [review]

**Pull request merged**